Runtime/BlockadeLabsSkyboxMesh.cs

```diff
@@ -123,7 +123,7 @@ internal GetImagineResult GetMetadata()
 
             if (renderer.sharedMaterial.mainTexture.name == "default_skybox_texture")
             {
-                new GetImagineResult() { request = new GetImagineRequest() { id = 0 } };
+                _ = new GetImagineResult { request = new ImagineResult { id = 0 } };
```

Comment:
.... I think this is supposed to return

Reply:
it was being discarded before, now it is explicit is all.